### PR TITLE
Document MongoDB admin config

### PR DESCRIPTION
**Pull request by ogeagla**
Aggregate info from:
- https://github.com/metabase/metabase/issues/9867
- https://github.com/metabase/metabase/issues/2722
- https://github.com/metabase/metabase/issues/3877

### CODE REVIEW

File: docs/administration-guide/databases/mongodb.md
Comment (mazameli) [inline docs/administration-guide/databases/mongodb.md]: > The resulting connection string has the relevant information to provide to the Metabase add a database for MongoDB admin page.

Would prefer "The resulting connection string has the relevant information to add to Metabase's `Add a Database` form for MongoDB."

File: docs/operations-guide/start.md
Comment (mazameli) [inline docs/operations-guide/start.md]: Maybe this item should go higher up in the list since it happens earlier in the process of getting Metabase up and running.

**Review by mazameli — CHANGES_REQUESTED** [review]
Left some style notes, but it might be a good idea to get another review from someone who can confirm the technical instructions (I have never connected to a Mongo db or set up SSL etc.).

File: docs/administration-guide/databases/mongodb.md
Comment (ogeagla) [reply docs/administration-guide/databases/mongodb.md]: I worked on a few MongoDB bugs when I started, and info kept pouring into those issues after the fact with what worked for folks, which is why I felt confident documenting this stuff.  I have either tried it myself several times for the MongoDB bugs, or some other time in the past and feel ok about these instructions.

File: docs/administration-guide/databases/mongodb.md
Comment (mazameli) [inline docs/administration-guide/databases/mongodb.md]: I don't think that docs are the right place for this type of "here's what's new" content; we usually try and do that in release notes. I think that the section below where you cover how to connect to Atlas clusters is sufficient for calling these changes out. I'd cut the "What's new in 0.32.9" bullet point and this paragraph because they won't age well.

File: docs/administration-guide/databases/mongodb.md
Comment (mazameli) [inline docs/administration-guide/databases/mongodb.md]: > Copy existing store and add self-signed cert to it:

I don't think you need this line anymore after the changes you made to the preceding sentence. Seems redundant.

**Review by mazameli — CHANGES_REQUESTED** [review]
Just two more text changes. I also just looked at the screenshots; if you send me the unannotated versions of all three of those, I can try marking them up a little more nicely if you don't mind. :)

**Review by mazameli — APPROVED** [review]
Looks good to go now from my end as long as my edited screenshots look good to you. :)

**Pull request merged**
